### PR TITLE
Expose libmc.decode_value

### PR DESCRIPTION
**Pull request by mckelvin**
`encode_value` and `decode_value` will be used for serialization in localcache.

cc @zzl0

### CODE REVIEW

File: libmc/_client.pyx [hunk 1]
```diff
@@ -282,11 +282,11 @@ cdef bytes _encode_value(object val, int comp_threshold, flags_t *flags):
 
 def encode_value(object val, int comp_threshold):
     cdef flags_t flags
-    cdef bytes = _encode_value(val, comp_threshold, &flags)
```
Comment (zzl0) [inline libmc/_client.pyx]: What's the type of `bytes` here before?

Comment (mckelvin) [inline libmc/_client.pyx]: That's depend on the return value of `_encode_value` (i.e.: the type of `bytes` is `bytes`). :sob:

Comment (zzl0) [inline libmc/_client.pyx]: :cool:

**Comment by zzl0** [comment]
:lgtm:

**Pull request merged**